geniza/corpus/admin.py

```diff
@@ -388,7 +395,9 @@ def merge_documents(self, request, queryset=None):
             status=303,
         )  # status code 303 means "See Other"
 
-    @admin.display(description="Export selected documents to CSV")
+    @admin.display(
+        description="Export selected documents to CSV", ordering="-shelfmark"
+    )  # @TODO: This ordering doesn't affect the order of the export because in the exporter we're ordering by ID
```

Comment:
ah, sorry — this is not the field I meant you to put the shelfmark ordering on; it should go on the `shelfmark_display` property on `Document`